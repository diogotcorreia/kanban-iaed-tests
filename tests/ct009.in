t 45 p
t 4 po
t 604 sbkjds
t 457290 heiow
l 1     3         2                                     4                       1
l          2       8               7       9   5
l       3
l 2         4
q