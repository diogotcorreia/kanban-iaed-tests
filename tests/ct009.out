task 1
task 2
task 3
task 4
1 TO DO #45 p
3 TO DO #604 sbkjds
2 TO DO #4 po
4 TO DO #457290 heiow
1 TO DO #45 p
2 TO DO #4 po
8: no such task
7: no such task
9: no such task
5: no such task
3 TO DO #604 sbkjds
2 TO DO #4 po
4 TO DO #457290 heiow
